src/toil/common.py

```diff
@@ -517,9 +519,12 @@ def _addOptionFn(*name, **kwargs):
     addOptionFn("--realTimeLogging", dest="realTimeLogging", action="store_true", default=False,
                 help="Enable real-time logging from workers to masters")
 
+    addOptionFn("--sse", dest="sse", default=False,
+                help="Enables server side encryption with the AWS job store. If --sseKey is not "
+                     "set, SSE (i.e. Amazon-managed encryption) will be used instead of SSE-C.")
     addOptionFn("--sseKey", dest="sseKey", default=None,
-            help="Path to file containing 32 character key to be used for server-side encryption on awsJobStore. SSE will "
-                 "not be used if this flag is not passed.")
+                help="Path to file containing 32 character key to be used for server-side encryption on "
+                     "awsJobStore. SSE will not be used if this flag is not passed.")
```

Comment:
For backwards compatibility a user should be able to only pass this argument & we should automatically set `config.sse = True`